swagger_spec_validator/validator20.py

```diff
@@ -21,5 +22,101 @@ def validate_spec_url(spec_url):
 
 
 def validate_spec(spec_json):
+    """Validates a Swagger 2.0 API Specification given a json dict.
+
+    :param spec_json: the json dict of the swagger spec.
+    :returns: `None` in case of success, otherwise raises an exception.
+    :raises: :py:class:`swagger_spec_validator.SwaggerValidationError`
+    :raises: :py:class:`jsonschema.exceptions.ValidationError`
```

Comment:
I don't think we should have consumers have to catch both of these.  We should catch `jsonschema.exceptions.ValidationError` and re-raise it as a SwaggerValidationError, passing along the relevant details from the error message